vagrant/README.md

```diff
@@ -18,7 +18,7 @@ Puppet will be used as the provisioning method in Vagrant and Hiera as the confi
 
     Additionally, you can copy a sample `config.yaml` file from the `vagrant-samples` folder found in puppet-<product> repository for quickly running a product on Vagrant.
 
-2. Download and copy Oracle JDK `1.8.0_112` distribution to the following path:
+2. Download and copy Oracle JDK `1.8.0_131` distribution to the following path:
 
     ````
     <PUPPET_HOME>/modules/wso2base/files/jdk-8u112-linux-x64.tar.gz
```

Comment:
Should be changed to 
<PUPPET_HOME>/modules/wso2base/files/jdk-8u131-linux-x64.tar.gz